acceptance-tests/upgrade/update_and_upgrade_dynamodb_test.go

```diff
@@ -14,11 +14,13 @@ var _ = Describe("UpgradeDynamoDBTest", Label("dynamodb", "upgrade"), func() {
 	When("upgrading broker version", func() {
 		It("should continue to work", func() {
 			By("pushing latest released broker version")
-			serviceBroker := brokers.Create(
+			options := []brokers.Option{
 				brokers.WithPrefix("csb-aws-dynamodb"),
 				brokers.WithSourceDir(releasedBuildDir),
 				brokers.WithReleaseEnv(),
-			)
+			}
+
+			serviceBroker := brokers.Create(append(options, extraEnvOptions...)...)
```

Comment:
I was thinking it might be cleaner to make `extraEnvOptions` a function. Then it wouldn't depend on a global variable.

Reply:
It can be a memoizing function, so that we won't need to read the files every single time.